src/jasmine-tests/bundle-stats/bs-add-ab-config-file-spec.js

```diff
@@ -32,11 +32,14 @@ describe("BundleStatsCollector - ", function() {
   var getAbConfig = function(ab) {
           return "AB.isOn('" + ab + "')";
       },
+  var getAbVariant = function(ab) {
+          return "AB.getVariant('" + ab + "')";
+      },
       ab1 = getAbConfig(string1),
       ab2 = getAbConfig(string2),
-      ab3 = getAbConfig(string3);
+      ab3 = getAbVariant(string3);
 
-  it("Adds ab configs to the collection.", function() {
+  it("Adds ab configs and variants to the collection.", function() {

```

Comment:
@thomas-huston-zocdoc I updated the test to cover AB variants. The README.MD suggests I need to install jasmine-node to run these tests. Is that accurate or is there an easier way? Can I run this repo through integration?